src/main/webapp/onelive/archive/index.html

```diff
@@ -154,6 +89,43 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
 <script>
     $(function () {
         loadNavAndFooter('/assets/content/static');  //relative path to content directory
+        loadYoutubeVideos();
     });
+
+    //function to load youtube videos
+    function loadYoutubeVideos() {
+        $.ajax({
+            type: 'get',
+            url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
+            dataType: 'json',
+            success: function (data) {
+                //mustache render
+                let renderedData = Mustache.render(
+                    $('#template-youtube-vidoe').html(), {data: data});
```

Comment:
"vidoe"

@Gravewalker666  @anjula-sack

Reply:
Oh! my fault. I must have missed that when I reviewed the code. Sorry for the mistake.